everrest-websockets/src/main/java/org/everrest/websockets/message/MessageSender.java

```diff
@@ -75,6 +80,7 @@ private void send(MessageWrapper message) throws IOException {
 
     private boolean isMaxQueueCapacityExceeded() {
         final int newSize = sendQueue.size() + 1;
+        LOG.debug(" SendQueue sieze {} ,  maxNumberOfMessageInQueue {}", sendQueue.size(), maxNumberOfMessageInQueue);
```

Comment:
Why you decided to place that logging here? In that case you will call that method on each message! It is not efficient IMO even in non-debug logging mode.